ee/models/assistant.py

```diff
@@ -0,0 +1,83 @@
+from collections.abc import Iterable
+
+from django.db import models
+from langgraph.checkpoint.serde.types import TASKS
+
+from posthog.models.team.team import Team
+from posthog.models.user import User
+from posthog.models.utils import UUIDModel
+
+
+class Conversation(UUIDModel):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    team = models.ForeignKey(Team, on_delete=models.CASCADE)
```

Comment:
Now, I think that there should be a ref to the project because property values can differ. @Twixes what do you think?